pkg/api/util/doget.go

```diff
@@ -51,6 +51,37 @@ func DoGet(c *http.Client, url string) (body []byte, e error) {
 
 }
 
+// DoGetExternal is a wrapper around performing HTTP GET requests designed for external endpoints.
+func DoGetExternal(c *http.Client, url string) (body []byte, e error) {
+	req, e := http.NewRequest("GET", url, nil)
+	if e != nil {
+		return body, e
+	}
+	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("Authorization", "Bearer "+GetAuthToken()) // To be updated for DCA token.
```

Comment:
I find it confusing that this function is called "DoGetExternal" it seems generic, but the bearer token is kinda specific

Reply:
There will be two tokens when using the DCA|DNA combo.
One for the internal API (the token can be gotten from GetAuthToken())
One for the External API which request will be validated with 
https://github.com/DataDog/datadog-agent/pull/1441/files#diff-0f2fdb9b159c3746841461f70dc08117R67.

I agree that DoGetExternal is very generic but I figured it would be used for the check scheduling too.

It's more of a method the agent will use to query different endpoints from the DCA specifically.

Given the above, what do you think ?